roles/zuul/meta/main.yml

```diff
@@ -1,2 +1,12 @@
 dependencies:
-  - pip
+  - role: python-app
+    name: nodepool
```

Comment:
I confirmed that this is why CI failed. It uses the name for venv, so zuul has /opt/venvs/nodepool, and everything gets confused from there.